ssg/build_profile.py

```diff
@@ -100,8 +102,8 @@ def get_profile_stats(self, profile):
                       % profile)
                 print("* Available profiles:")
                 profiles_avail = self.tree.findall("./{%s}Profile" % (xccdf_ns))
-                for profile in profiles_avail:
-                    print("** %s" % profile.get('id'))
+                for _profile in profiles_avail:
+                    print("** %s" % _profile.get('id'))
```

Comment:
why?

Reply:
`profile` is shadowed from the outer scope, and `pylint` doesn't like it. This is technically a false-positive since the `sys.exit` call immediately following this loop ensures that the changes to `profile` won't get seen, but that's a different matter. See issue on line 105 on [Scrutinizer](https://scrutinizer-ci.com/g/OpenSCAP/scap-security-guide/inspections/a6da40b5-2622-4920-8a8a-d684fb475803/issues/files/ssg/build_profile.py?status=all&orderField=path&order=asc&honorSelectedPaths=0#inspectioncomment-106230593).

Reply:
(I can drop that change if you want).